src/Alert/index.js

```diff
@@ -0,0 +1,151 @@
+// @flow
+import * as React from "react";
+import * as tokens from "@kiwicom/orbit-design-tokens";
+
+import * as Icons from "../icons";
+import { Button } from "../index";
+
+const Icon = ({ type, Icon: CustomIcon, customColor }) => {
```

Comment:
What is doing `CustomIcon` ?

Reply:
`CustomIcon` is there for a case if you want to choose the custom icon, different from default one.